pkg/where-resolver/reconcile_on_edgeplacement.go

```diff
@@ -87,15 +86,14 @@ func (c *controller) reconcileOnEdgePlacement(ctx context.Context, epKey string)
 	singles := []edgev2alpha1.SinglePlacement{}
 	for _, loc := range locsFilteredByEp {
 		// 2)
-		lws := logicalcluster.From(loc)
 		stsInLws, err := c.synctargetLister.List(labels.Everything())
```

Comment:
This variable name should have been changed in #1265 , it now holds a list of _all_ SyncTarget projections into the KCS. @waltforme

Reply:
done